filebeat/module/kibana/log/ingest/pipeline.yml

```diff
@@ -37,18 +37,12 @@ processors:
     field: kibana.log.meta.res.statusCode
     target_field: http.response.status_code
     ignore_missing: true
-- rename:
-    field: kibana.log.meta.res.responseTime
-    target_field: temp.duration
-    ignore_missing: true
 - script:
     lang: painless
-    source: ctx.event.duration = Math.round(ctx.temp.duration * params.scale)
-    params:
-      scale: 1000000
```

Comment:
I think that the use of params was needed to avoid recompilations of pipelines, but I am not sure of the specifics :thinking: 
@andrewkroh do you remember more details on this?

Reply:
I believe `params` in unnecessary in this context since the scaling factor is essentially a constant and we don't reuse the script anywhere. I think this guidance originally came from an interpretation of https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-scripting-using.html#prefer-params.

Reply:
Params are useful if you have the same script that runs with _different_ params.  They do not save compilations if you're embedding a constant.

Reply:
> > @jsoriano I'm seeing what appears to be multiple errors in the windows 10 build test but none of them seem related to my changes. Any thoughts?
> 
> This is weird because last build actually passed https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats/detail/PR-25675/8/pipeline/, probably the github checks haven't been properly updated.

so is this good to merge??

Reply:
Thanks for the clarifications, all good then :slightly_smiling_face: